test/template-service-test.js

```diff
@@ -0,0 +1,123 @@
+'use strict';
+
+// Disable self signed testing for the tests
+var service = new (require('../service/template-service'))();
+var fs = require('fs');
+
+var templateModels = {
+  validTemplate: './test/template/test-email.html',
+  invalidTemplate: './test/template/' + global.testUid() + '/test-email.html',
+  templatePath: './test/template',
+  loadedTemplates: ['test-email.html'],
+  createTemplate: './test/template/' + global.testUid() + '-email.html',
+  templateBody: '<h1>Hello {{name}}</h1>',
+  updateTemplateBody: '<h1>Hello {{name}}</h1>This is updated',
+};
+
+describe('The service should be initialised', function serviceInitialization() {
```

Comment:
you wouldn't normally worry about naming anon function expression in your tests because if there is an error, the assertion lib will throw and indicate where in the code the error is. Named functions are useful for finding an error in a trace where there isn't an assertion being thrown.